lib/Notes/NotePopupModal/NotePopupModal.js

```diff
@@ -0,0 +1,182 @@
+import {
+  useState,
+  useEffect,
+  useCallback,
+} from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+// eslint-disable-next-line import/no-webpack-loader-syntax
+import '!style-loader!css-loader!react-quill/dist/quill.snow.css';
```

Comment:
FYI, filed [STRWEB-16](https://issues.folio.org/browse/STRWEB-16), "Configure loaders to avoid webpack loader syntax elsewhere".